### PR TITLE
[4.0] Adding accessibility settings to admin profile view

### PR DESCRIPTION
**Pull request by Hackwar**
Pull Request for Issue #30217.

### Summary of Changes
The accessibility settings are missing from the profile view of com_admin. This copies those over from the com_users XML.

### Testing Instructions
In the backend in the upper right corner click on the user menu and select accessibility settings. Notice that the new page does not contain accessibility settings. Apply the patch, enjoy the new settings.

### CODE REVIEW

File: administrator/components/com_admin/forms/profile.xml [hunk 1]
```diff
@@ -152,6 +152,49 @@
...
 		</fieldset>
+		<!-- User accessibility settings -->
+		<fieldset name="accessibility" label="COM_ADMIN_A11Y_SETTINGS_FIELDSET_LABEL">
+			<field
+					name="a11y_mono"
+					type="radio"
+					label="COM_ADMIN_A11Y_SETTINGS_FIELD_MONOCHROME"
+					layout="joomla.form.field.radio.switcher"
+					default="0"
+			>
```
Comment (sandewt) [inline administrator/components/com_admin/forms/profile.xml]: `>`  greater then sign:

```
default="0"
	>
		<option value="0">JNO</option>
		<option value="monochrome">JYES</option>
	</field>
```
should be, 

```
default="0"
	        >
		<option value="0">JNO</option>
		<option value="monochrome">JYES</option>
	</field>
```

and so on...

**Comment by ceford** [comment]
I have tested this item :white_check_mark: successfully on e8004fae1c0b4f95e74ea4e458a919a94efdd54e<br /><br />It works. But - It seems that accessibility setting must be on or off for the site by enabling the plugin. It would be much better to enable it by individual user - here.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31884">issues.joomla.org/tracker/joomla-cms/31884</a>.</sub>

**Comment by toivo** [comment]
I have tested this item :white_check_mark: successfully on e8004fae1c0b4f95e74ea4e458a919a94efdd54e<br /><br />Tested successfully in Beta7-dev of 8 January.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31884">issues.joomla.org/tracker/joomla-cms/31884</a>.</sub>

**Comment by brianteeman** [comment]
>  It would be much better to enable it by individual user - here.

Why

File: administrator/components/com_admin/forms/profile.xml (hunk 1 above)
Comment (Hackwar) [reply administrator/components/com_admin/forms/profile.xml]: fixed.

**Comment by ceford** [comment]
> > It would be much better to enable it by individual user - here.
> 
> Why

Did I miss something? Are the Accessibility settings on site-wide for everybody or off for everybody? I just expect it to be a personal option.

**Comment by chmst** [comment]
These settings are only for backend users and only for the backend.

**Comment by brianteeman** [comment]
It is a personal option if you use the settings. 
It is a site owners option if they are available to those backend users (enable/disable the plugin)

Think about it carefully and you will "get it"

**Comment by wilsonge** [comment]
> It is a site owners option if they are available to those backend users (enable/disable the plugin)

Not in this case @brianteeman - this PR is dealing with the template settings. It's got nothing to do with the plugin

**Comment by brianteeman** [comment]
Sorry I got confused by @ceford unrelated comment

**Comment by Quy** [comment]
Is PR #29226 not valid?

**Comment by Hackwar** [comment]
I don't care which PR is used. I want this Release Blocker fixed and right now I would use the quickest way possible. This is not something that we can't change in Joomla 4.1 or later.

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**